src/current/v23.2/column-level-encryption.md

```diff
@@ -0,0 +1,331 @@
+---
+title: Column Level Encryption
+summary: SQL users are required to pass encryption keys when reading from or writing to specific columns
+toc: true
+docs_area: reference.sql
+---
+
+CockroachDB v23.2 and later support _column level encryption_. Using this feature, you can encrypt one or more of the columns in each row of a database table.
+
+Column level encryption can be useful for compliance scenarios such as adhering to PCI or GDPR.
+
+Once a column is encrypted, SQL users are required to pass an encryption key when reading from or writing to that column. This prevents users without the encryption key from taking actions like the following:
+
+- Reading sensitive information (like _personally identifiable information_ ([PII](https://en.wikipedia.org/wiki/Personal_data))) when accessing the tables.
+- Adding or updating records without encrypting the specified columns.
```

Comment:
Nothing stops them for inserting garbage data into a column since the column type will just be bytes.

Reply:
So are you saying we should delete the second bullet? Since in fact there is no prevention from updating records without encrypting them, they can just smash some random bytes in there?  

If that's the case, I can update this to kill the bulleted list and move the 'reading sensitive information...' back into the preceding paragraph

Reply:
Yep that sounds good!

Reply:
OK just made this update. If it looks good to you, give the ✅  and I'll move this along to docs team review and then merge!